claim/gql_mutations.py

```diff
@@ -311,6 +320,11 @@ def update_or_create_claim(data, user):
     return claim
 
 
+def __autogenerate_claim_code():
+    from .utils import __autogenerate_nepali_claim_code
```

Comment:
Please use absolute imports, also methods marked as `__private` shouldn't be imported.

Reply:
Ok, if I move this code to another place as suggested in the next comment, what do you propose? Protected method? Or just a public one?

Reply:
I'd just change __autogenerate_nepali_claim_code to public